janusgraph-codepipelines-ci/README.md

```diff
@@ -72,44 +72,68 @@ per region per account. If you need more, you can
 Here is an example of a file that defines two pipelines with parallel build actions.
 
 ```yaml
+defaultComputeImage: stephenreed/jenkins-java8-maven-git:latest
```

Comment:
True. git is avoided be cause the source step earlier in the pipeline does the pulling bit from git. good catch!